Resources/Prototypes/_NF/Entities/Structures/Machines/shredder.yml

```diff
@@ -0,0 +1,33 @@
+- type: entity
+  id: Shredder
+  parent: BaseLathe
+  name: shredder
+  description: It shreds things. What more is there to say?
+  components:
+  - type: Sprite
+    sprite: _NF\Structures\Machines\Shredder.rsi
```

Comment:
while this likely still works in game, it makes the yaml linter cry, so I'll switch these to forward slashes instead